src/game.cpp

```diff
@@ -120,6 +120,18 @@ bool is_valid_in_w_terrain(int x, int y)
     return x >= 0 && x < TERRAIN_WINDOW_WIDTH && y >= 0 && y < TERRAIN_WINDOW_HEIGHT;
 }
 
+/*
+ * case insensitive string::find( string::findstr ). findstr must be lowercased
+ */
+bool lcmatch(const std::string &str, const std::string &findstr)
```

Comment:
Well, it doesn't feel like UI.  It's string function. Should I make new header or there more appropriate header?